events/pr-listener.js

```diff
@@ -0,0 +1,68 @@
+const { Client, GatewayIntentBits } = require('discord.js');
+const axios = require('axios');
+const { differenceInDays } = require('date-fns');
```

Comment:
Are we only using this library to subtract two dates? Im a bit partial to just doing a quick function for this rather than installing a package. Should be straight forward?

Reply:
Also moving away from difference in days and doing like absolute time or seconds or smt would make it easier to have a more precise amount of time, useful if I want like 10 mins for testing or smt.